rust/geoparquet/src/metadata.rs

```diff
@@ -454,73 +454,6 @@ pub struct GeoParquetMetadata {
     pub columns: HashMap<String, GeoParquetColumnMetadata>,
 }
 
-/// GeoParquet column metadata
-#[derive(Clone, Debug, Serialize, Deserialize)]
-pub struct GeoParquetColumnMetadata {
```

Comment:
This is just moved down so that `impl` blocks can be as close as possible to their associated struct definitions